docs_src/design/adr/core/0022-UoM.md

```diff
@@ -0,0 +1,206 @@
+
+The unit of measure was, however, in all cases just a string reference to some arbitrary unit (which may or may not be in a UoM standard) to be interpreted by the consumer of EdgeX data.  The reporting sensor/device or programmer of the device service could choose what UoM string was associated to the device resources (and readings produced by the device service) as the unit of measure for any piece of data.  Per the temperature example above, the unit of measure could have been "F" or "C", "Celsius" or "Fahrenheit", or any other representation.  In other words, the associated unit of measure for all data in EdgeX was left to agreement and interpretation by the data provider/producer and EdgeX data consumer.
+
+There are various specifications and standards around unit of measure.  Specifically, there are several options to choose from as it relates to the exchange of data in electronic communications - and units of measure associated in that exchange.  As examples, two big competing standards around EDI (electronic data exchange) that both have associated unit of measure codes are:
+
+- [ANSI X12: EDI standard used mostly in the US](https://ediacademy.com/blog/x12-unit-of-measurement-codes/)
+- [EDIFACT: UN EDI standard used mostly in Europe and Asia](https://unece.org/fileadmin/DAM/cefact/recommendations/rec20/rec20_rev3_Annex2e.pdf)
+
+The [Unified Code for Units of Measure](https://en.wikipedia.org/wiki/Unified_Code_for_Units_of_Measure) provides an alternative list (not a standard) that is used by various organizations like OSGI and the Eclipse Foundation.
+
+While standards exist, use by various open source projects (especially IoT/edge projects) is inconsistent and haphazard.   Groups like oneM2M seem to define their own selection of units in specifications per vertical (home for example) while Kura doesn't even appear to use the UoM JSR (a Java related unit of measure specification for Java applications like Kura).
+
+## Decision
+
+It would be speculative and inappropriate for EdgeX to select a unit of measure standard which is not widely adopted in the industry or choose a static unit of measure list that is incomplete with regard to possible IoT / edge use case needs.  At this time, there does not appear to be a single and unequivocal standard for units of measure that encompasses all EdgeX related use cases (now and in the future). 
+
+Therefore, EdgeX chooses not to select or adopt a unit of measure specification, standard, or code list to apply across the platform.  Instead, EdgeX adopters will be allowed to optionally specify which unit of measure specification, standard, or unit of measure code list they would like used in their instance(s) of EdgeX.
```

Comment:
For reference, the ability to load units registry from a URI (which is now removed) was originally suggested to address the mentioned problem of not being able to find a single set of units that satisfies all use cases. The above text implies that this is indeed a requirement which EdgeX could not meet due to that reason.

Instead, EdgeX could have picked one good set (e.g. ANSI X12 units) and offered it publicly as a TOML file (loadable via a URL), while the community could add others for public (URL) or private use (File URI or relative path). This would have made it possible to set the units registry via an environment variable, without the need to statically deploy or package the file.

Reply:
discussed in Core WG of 3/3/22.  Removed the idea of inline UoM definition to keep things simple for the first iteration.  Also removed URI reference or "global configuration" concept until this can be explored in future planning/architect's meeting.